cos-fleetshard-operator-debezium/src/main/java/org/bf2/cos/fleetshard/operator/debezium/DebeziumOperandController.java

```diff
@@ -217,4 +224,10 @@ public boolean delete(ManagedConnector connector) {
 
         return kctr && kc && secret;
     }
+
+    private String getOperatorName(ManagedConnector connector) {
+        return connector.getStatus().getConnectorStatus().getAssignedOperator().getId() +
+            "-" +
+            connector.getStatus().getConnectorStatus().getAssignedOperator().getVersion();
```

Comment:
is this correct ? I recall the in latest discussion I had with @valdar, the operator id would be set to a unique value in the deployment as an env var.

Reply:
Hi @lburgazzoli, this works locally but this is likely due to what is set as the default values in the `kustomization.yaml` inside of the `/etc/kubernetes/operator-<operator>/local/` directories.  Let me know if this should be changed to something else.

Reply:
Maybe we can make those configurable from `application.properties`?

Reply:
@Naros yes they need to be changed in the repo, likely the operator id need to be changed i.e. to something unique (looks at the addon/bundle)
@rk3rn3r it is already the case, the id and version are from an env var/property

Reply:
@lburgazzoli yes, but this code makes the assumption that there is this pattern to generate the label from the operator id and version. A config option that just sets the full label name is more flexible and avoids code changes when the pattern changes.

Reply:
That label should just use the operator id, it should not concatenate things (see my first comment)

Reply:
Gotcha now! Sorry!